style.css

```diff
@@ -0,0 +1,406 @@
+  background-color: var(--white-bg-color);
+}
+
+#hamburger.active .bar:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+  background-color: var(--white-bg-color);
+}
+
+/* .menu-icon {
+  width: 24px;
+  height: 24px;
+  margin-right: 12px;
+  cursor: pointer;
+} */
+
+section {
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: 114px 24px;
+  gap: 12px;
+}
+
+ul {
+  list-style: none;
+}
+
+.headline {
+  background-image: url("image/header-shapes-mobile.png");
+  border-bottom-left-radius: 60px;
+}
+
+.title {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.details {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.contact-link {
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.social-media {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.social-media-links {
+  display: flex;
+  gap: 20px;
+  text-decoration: none;
+}
+
+.works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 88px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  gap: 12px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.project-info,
+.project-header {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  gap: 12px;
+}
+
+.project-title {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.header-details {
+  display: flex;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+}
+
+.header-details .client {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #172b4d;
+}
+
+.header-details .role,
+.header-details .year {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.separator-circle {
+  width: 8px;
+  height: 8px;
+  background: #c1c7d0;
+  border-radius: 50%;
+}
+
+.project-details {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.tags {
+  display: flex;
+  gap: 8px;
+  align-items: center;
+}
+
+.tags .tag {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  color: #6070ff;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+}
+
+.action {
+  padding: 12px 0;
+}
+
+.btn-default {
+  width: fit-content;
+  padding: 10px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  font-family: inherit;
+  font-style: normal;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background: transparent;
+  color: #396df2;
+  cursor: pointer;
+  transition: all 0.3s ease-in-out;
+}
+
+.btn-default:hover {
+  background: #6070ff;
+  color: #fff;
+}
+
+.about {
+  border-top-right-radius: 60px;
+}
```

Comment:
- [ ] KIndly make sure to align your `about` section content with CSS `flexbox` as it is one mandatory project requirement. :wink: This will help you to maintain website responsiveness. :100:

Reply:
Done.